7_days_to_die/egg-7-days-to-die.json

```
@@ -32,103 +33,171 @@
     },
     "variables": [
         {
-            "name": "Max Players",
-            "description": "Maximum Concurrent Players",
-            "env_variable": "MAX_PLAYERS",
-            "default_value": "8",
+            "sort": 7,
+            "name": "Auto Update",
+            "description": "This is to auto update the server on start.\r\n\r\nOptions are 0 or 1\r\nDefault is 1",
+            "env_variable": "AUTO_UPDATE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": [
+                "required",
+                "boolean"
+            ]
+        },
+        {
+            "sort": 4,
+            "name": "Crossplay Support",
+            "description": "Enables\/Disables crossplay",
+            "env_variable": "CROSSPLAY_SUPPORT",
+            "default_value": "false",
             "user_viewable": true,
             "user_editable": false,
-            "rules": "required|string|max:20",
-            "sort": null,
-            "field_type": "text"
+            "rules": [
+                "required",
+                "string",
+                "in:true,false"
```

```suggestion
            "rules": [
                "required",
                "boolean"
```
https://laravel.com/docs/12.x/validation#rule-boolean